parent/pom.xml

```diff
@@ -1066,53 +1070,141 @@
 			</properties>
 		</profile>
 
+		<!-- JBIDE-23206
+			 To force a local build of a pull request to use current date/time when generating plugins and features,
+			 instead of using jgit timestamps (which could be in the past, and therefore result in stuff that Eclipse
+			 thinks is OLDER that a current N build, use this "R" profile:
+			 mvn verify -PR
+		-->
 		<profile>
-			<id>test-on-installed-app</id>
+			<id>R</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-packaging-plugin</artifactId>
+						<version>${tychoVersion}</version>
+						<configuration>
+							<timestampProvider>default</timestampProvider>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		
+		
+		<!-- IT Tests profiles -->
+		<profile>
+			<id>it-tests</id>
+			<properties>
+				<surefire.timeout>7200</surefire.timeout>		
+			</properties>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>com.googlecode.maven-download-plugin</groupId>
+						<artifactId>download-maven-plugin</artifactId>
+						<executions>
+							<execution>
+								<id>download-pluginCustomization</id>
+								<phase>pre-integration-test</phase>
+								<goals>
+									<goal>wget</goal>
+								</goals>
+								<configuration>
+									<url>https://raw.githubusercontent.com/jbosstools/jbosstools-integration-tests/master/tests/pluginCustomization.ini</url>
```

Comment:
Since this file is only needed by integration tests for jbosstools, why would you want it anywhere but in github.com/jbosstools/jbosstools-integration-tests ?

What's wrong with https://raw.githubusercontent.com/jbosstools/jbosstools-integration-tests/master/tests/pluginCustomization.ini ?